src/prepack-cli.js

```diff
@@ -42,6 +43,23 @@ let flags = {
   delayUnsupportedRequires: false,
   internalDebug: false,
 };
+
+const compatibilityDict: { [string]: ?Compatibility } = {};
+Object.values(CompatibilityValues).map((val: any) => { compatibilityDict[val] = val; })
+
+function parseCompatibility(str: string): Compatibility {
+
+  const compatibility = compatibilityDict[str];
```

Comment:
If `CompatibilityValues` you could just do `CompatibilityValues.includes(str)` here.